applications/launcher/eventfilter.cpp

```diff
@@ -34,18 +39,33 @@ QPointF globalPos(QQuickItem* obj){
     }
     return QPointF(x, y);
 }
-QMouseEvent* toMouseEvent(QEvent::Type type, QEvent* ev){
+
+QMouseEvent* tabletToMouseEvent(QEvent::Type type, QEvent* ev){
     auto tabletEvent = (QTabletEvent*)ev;
     auto button = tabletEvent->pressure() > 0 || type == QMouseEvent::MouseButtonRelease ? Qt::LeftButton : Qt::NoButton;
     return new QMouseEvent(
-        type,
-        transpose(tabletEvent->posF()),
-        transpose(tabletEvent->globalPosF()),
-        transpose(tabletEvent->globalPosF()),
-        button,
-        button,
-        tabletEvent->modifiers()
-    );
+                type,
```

Comment:
You've changed the indentation here instead of matching the surrounding code style.